src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+  const partnerType = (sex === 'm') ? 'wife' : 'husband';
```

Comment:
in ternary you have to place each part of it on the new line, and this braces aren't necessity